test/test_list.rb

```diff
@@ -1,7 +1,10 @@
 # NOTE: following now done in helper.rb (better Readability)
 require 'helper'
 
-ActiveRecord::Base.establish_connection(adapter: "sqlite3", database: ":memory:")
+ActiveRecord::Base.establish_connection(
+  adapter: "sqlite3",
+  database: 'file:memdb1?mode=memory&cache=shared'
```

Comment:
I guess this is nice to have, but outside the scope of this PR. Does anything in this PR break w/o that change?